src/rpc.ts

```diff
@@ -14,7 +15,7 @@ router.post('/', async (req, res) => {
       await send('subscribe', {
         to: params.email,
         address: params.address,
-        name: 'Fabien'
+        name: constants.example.name
```

Comment:
I think we wont have name when someone subscribe we can just remove this parameter